telosnetwork/telos-wallet
# bug - low ram evm account creation and bridge transactions

# Fixes #419 #314 

## Description
- See issue descriptions

## Test scenarios

1. Select 'Native' and create a new account (be sure to save keys)
2. Add Newly created account to Anchor
3. Sign in to 'Native' using Anchor and newly created account
4. click '>>> EVM' to initiate a deposit to EVM and display deposit form
5. click 'Generate New Address' 
6. see updated error instructing to view resources
7. Seed new account with 1 TLOS from external wallet (Anchor, etc)
8. return to wallet and purchase .25 TLOS worth of RAM
9. retry address generation, see success
10. Login to OBE with new account
11. Sell all available ram (after linked EVM account creation)
12. Return to web wallet
13. view resources to verify 100% ram usage
14. repeat step 4, and either send to linked address, or other address
15. see success and fuel message in Anchor transaction

## Checklist:
